src/librustc_back/target/apple_base.rs

```diff
@@ -47,6 +47,7 @@ pub fn opts() -> TargetOptions {
         exe_allocation_crate: super::maybe_jemalloc(),
         has_elf_tls: version >= (10, 7),
         abi_return_struct_as_int: true,
+        emit_debug_gdb_scripts: false,
```

Comment:
Is this really right? Does no one use gdb on macOS?

Reply:
This doesn't directly answer your question but this is equivalent to the current logic. See the diff on `src/librustc_trans/debuginfo/gdb.rs`; it has `!cx.sess().target.target.options.is_like_osx`. So I assume the answer is that people mainly use lldb?

Reply:
Ah, this makes sense!

Reply:
@whitequark Locally gdb is a pain to use on macOS because it would need to be signed in order to have access to process information and no-one cares to actually sign it and it's not shipped with macOS. It is "usable" for target != host but not necessarily the best option.